src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -13,6 +13,50 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
 {
     public class CodeGenOperatorTests : CSharpTestBase
     {
+
+        [Fact]
+        public void TestIsNullPattern()
```

Comment:
Other tests that we should add:

- Value of unconstrained type parameter testing `is null` 
- Value of type parameter constrained to `class` against `is null`
- A constant, which is null, testing against `is null`